script.module.clouddrive.common/clouddrive/common/ui/utils.py

```diff
@@ -95,7 +98,8 @@ def execute_json_rpc(method, params=None, request_id=1):
         cmd = {'jsonrpc': '2.0', 'method': method, 'id': request_id}
         if params:
             cmd['params'] = params
-        return json.loads(xbmc.executeJSONRPC(json.dumps(cmd)))
+        cmd = json.dumps(cmd)
+        return json.loads(xbmc.executeJSONRPC(cmd)) 
     
```

Comment:
This is prone to errors. Try to catch the exception

Reply:
The try-catch is always in the main method calling this util method. I don't want to eat the exception or handle it here because each method calling this one will handle it.